doc/lua_api.txt

```diff
@@ -2535,13 +2535,10 @@ Some types may inherit styles from parent types.
     * textcolor - color. Default white.
 * label, vertlabel
     * noclip - boolean, set to true to allow the element to exceed formspec bounds.
-* image_button
-    * alpha - boolean, whether to draw alpha in bgimg. Default true.
-    * border - boolean, draw border. Set to false to hide the bevelled button pane. Default false.
-    * noclip - boolean, set to true to allow the element to exceed formspec bounds.
-* item_image_button
-    * border - boolean, draw border. Set to false to hide the bevelled button pane. Default false.
-    * noclip - boolean, set to true to allow the element to exceed formspec bounds.
+* image_button (additional properties)
+    * fgimg - standard image. Defaults to none.
```

Comment:
State which image take precedence (it should probably be the element one)

Reply:
Good catch actually, the element-supplied textures were not taking precedence but should've been. Other style properties were already subject to this, but I missed those two.